grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovement.groovy

```diff
@@ -229,6 +230,19 @@ class OutboundStockMovement implements Serializable, Validateable {
         return shipment?.currentStatus == ShipmentStatusCode.RECEIVED
     }
 
+    @Deprecated
+    Map<String, String> getDisplayStatus() {
+        StockMovementContext stockMovementContext = new StockMovementContext(
+                order: order,
+                requisition: requisition,
+                shipment: shipment,
+                origin: origin,
+                destination: destination
```

Comment:
I'm wondering if it was safe to remove the `defaultStatus` fallback. Wouldn't it be safer to have the fallback to current `status` property of the `OutboundStockMovement`, as it would already have the context determined and an expected status, instead of "hard-coding" the default status as `shipment.status ?: requisition.status`
Unless we intentionally want to remove it and catch all the regression easier instead of having "a safe fallback"?

Reply:
Correct me if I am wrong but for all 3 cases 
- `outboundStockMovement`
- `StockMovement`
-  `OutboundStockMovementListItem`
 **defaultStatus** is a requisition status that is returned as a fallback in the `StockMovementStatusResolver` getStatus methods

Reply:
yeah, you currently implemented it like this, but with the `defaultStatus` version, I did a fallback to `status` which was something else depending on the DTO you've mentioned.
So for the `OutboundStockMovement` `status` is not really the requisition status (at least not always), but it is determined in the `stock-movement.sql` view - as for regular SMs it is apparently the requisition status, but for returns it is mapped to something else.
We should be safe with your version anyway, I just wanted to explain the purpose of the `defaultStatus` - so its purpose was to (if something goes wrong) fallback to the status, that was returned before out refactor.